lib/config/env.js

```diff
@@ -39,5 +39,36 @@ function getConfig(env) {
     }
   });
 
+  if (env.GITHUB_COM_TOKEN) {
+    config.endpoints.push({
+      platform: 'github',
+      endpoint: 'github.com',
```

Comment:
Until now we use a full URL syntax like `https://api.github.com` for `endpoint`. Can we keep that?